### PR TITLE
added christmas 2021 landing page

### PR DESCRIPTION
**Pull request by michellengnx**

### CODE REVIEW

**Review by Andrewnt219 — CHANGES_REQUESTED** [review]
@michellengnx looks great 👍. Some content is missing like the text before the button, and the font-size + font-weight is different from the design.

File: src/pages/christmas-2021.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import MainLayout from "@components/pages/MainLayout";
```
Comment (Andrewnt219) [inline src/pages/christmas-2021.tsx:1]: Can you make this page a component and temporarily replace the current hero on homepage with this?

**Review by Andrewnt219 — CHANGES_REQUESTED** [review]

File: src/pages/christmas-2021.tsx (hunk 1 above)
Comment (michellengnx) [reply src/pages/christmas-2021.tsx:1]: Will do!

File: src/pages/index.tsx [hunk 2]
```diff
@@ -93,48 +92,132 @@ const Index: VFC<Props> = ({ data, error }) => {
...
   return (
-    <section
-      tw="col-span-full grid grid-cols-12 content-start xl:-mt-24"
-      aria-labelledby="hero-title"
-    >
-      <header tw="text-center grid-p-sm  xl:(col-start-2 col-end-6 self-center z-10 width[150%] text-left)">
-        <h1
-          id="hero-title"
-          tw="text-2xl text-primary font-black md:text-4xl lg:text-5xl"
+    <section css={"width: 100vw"}>
```
Comment (Andrewnt219) [inline src/pages/index.tsx]: This causes an overflow on desktop.
```suggestion
    <section tw="col-span-full">
```

**Review by Andrewnt219 — CHANGES_REQUESTED** [review]

**Review by Andrewnt219 — APPROVED** [review]

**Pull request merged**